inabox/tests/integration_test.go

```diff
@@ -20,15 +20,20 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func mineAnvilBlocks(t *testing.T, numBlocks int) {
+func mineAnvilBlocks(t *testing.T, testHarness *TestHarness, numBlocks int) {
 	t.Helper()
 	for i := 0; i < numBlocks; i++ {
-		err := rpcClient.CallContext(t.Context(), nil, "evm_mine")
+		err := testHarness.RPCClient.CallContext(t.Context(), nil, "evm_mine")
 		require.NoError(t, err)
 	}
 }
 
 func TestEndToEndScenario(t *testing.T) {
+	// Create a fresh test context for this test
+	testCtx, err := NewTestHarnessWithSetup(globalInfra)
```

Comment:
why call this testCtx instead of testHarness? Its not a ctx right?

Reply:
4b937c0748862cef91cb6aa74ab6e4745383d157